tests/test_TFNetworkLayer.py

```diff
@@ -150,6 +150,137 @@ def test_concat_sources_missing_dim():
     session.run(out.output.placeholder)
 
 
+def test_self_att_time_dim_tags():
+  with make_scope() as session:
+    n_in = 128
+    channels = 16
+    n_time = 128
+    n_batch = 10
+    context_size = 15
+    rnd = numpy.random.RandomState(42)
+    querry_dim = 32
+    querry_dim += context_size
+    key_dim = 32
+    value_dim = 32
```

Comment:
Use different numbers for `querry_dim`, `key_dim` and `value_dim`.

Reply:
Done